examples/js/loaders/GLTF2Loader.js

```diff
@@ -2475,6 +2464,47 @@ THREE.GLTF2Loader = ( function () {
 
 					}
 
+					// for Specular-Glossiness
+					if ( child.material && child.material.type === 'ShaderMaterial' ) {
```

Comment:
@takahirox Can you change this so `material.type` is not used for this purpose? I do not think that pattern is used elsewhere...

In this case, it should be something like
```
if ( child.material && child.material.isSpecularGlossinessMaterial ) {
```